src/renderer/services/chain/fees/deposit.ts

```diff
@@ -94,10 +94,10 @@ const depositFeeByChain$ = (chain: Chain, type: DepositType): FeeLD => {
           )
         )
       )
-    case 'ETH':
-      return Rx.of(RD.failure(new Error('Deposit fee for ETH has not been implemented')))
```

Comment:
Pls don't remove it ^. Instead of, we should do same `failure` for chains we don't have implemented yet.